README.md

```diff
@@ -1,2 +1,395 @@
+   Storage locations available ([see this link for full descriptions](https://curc.readthedocs.io/en/latest/compute/filesystems.html)):
+
+   - __Home filesystem:__ 2 GB of backed up space under `/home/$USER` (where `$USER` is your RMACC or Alpine username).
+   - __Projects filesystem:__ 250 GB of backed up space under `/projects/$USER` (where `$USER` is your RMACC or Alpine username).
+   - __Scratch filesystem:__ 10 TB (10,240 GB) of space __*which is not backed up*__ under `/scratch/alpine/$USER` (where `$USER` is your RMACC or Alpine username).
+
+1. __External / remote storage:__ Users are encouraged to explore external data storage options for long-term hosting.<br>
+   Examples may include the following:
+
+   - __[PetaLibrary](https://www.colorado.edu/rc/resources/petalibrary)__: subsidized external storage host from University of Colorado Boulder's Research Computing (requires specific arrangements outside of Alpine).
+   - __Cloud hosting:__ [object storage](https://en.wikipedia.org/wiki/Object_storage) and related data hosting options from cloud providers like [Microsoft Azure](https://azure.microsoft.com/en-us), [Google Cloud](https://cloud.google.com/) ([internal CU Anschutz GC information](https://www.cuanschutz.edu/offices/office-of-information-technology/tools-services/google-cloud-platform)), or [Amazon Web Services](https://aws.amazon.com/).
+   - __Others:__ additional options include third-party "storage as a service" offerings like Google Drive or Dropbox and/or external servers maintained by other groups.
+
+### How do I send or receive data on Alpine?
+
+```mermaid
+flowchart LR
+    external_storage["📁 External Storage\n(user specified / configured)"]
+    subgraph alpine["🖥️ Alpine"]
+        subgraph storage["📂 Storage"]
+            local_storage["📁 Local Storage\n(sometimes temporary)"]
+        end
+    end
+
+    external_storage --> | send data\nto Alpine | local_storage
+    local_storage --> | receive data\nfrom Alpine | external_storage
+
+style alpine fill:#ffffff,stroke:#444444;
+```
+
+_Diagram showing external data storage being used to send or receive data on Alpine local storage._
+
+Data may be sent to or gathered from Alpine using a number of different methods.
+These may vary contingent on the external data storage being referenced, the code involved, or your group's available resources.
+Please reference the following documentation from the University of Colorado Boulder's Research Computing regarding data transfers: [The Compute Environment - Data Transfer](https://curc.readthedocs.io/en/latest/compute/data-transfer.html).
+__Please note:__ due to the authentication configuration of Alpine many local or SSH-key based methods are not available for CU Anschutz users.
+As a result, [Globus](https://www.globus.org/) represents one of the best options available (see [3. 📂 Transfer data results](#3-%F0%9F%93%82-transfer-data-results) below).
```

Comment:
Awesome; AFAIK Globus is (currently) the recommended mechanism for getting data in and out of Alpine, and it's great to see your detailed tutorial on it later in the text. It may be worth making special note here that Globus can be used both to transfer data into Alpine and out of it, e.g.:

```suggestion
As a result, [Globus](https://www.globus.org/) represents one of the best options available (see [3. 📂 Transfer data results](#3-%F0%9F%93%82-transfer-data-results) below). While the Globus tutorial in this document describes how you can download data from Alpine to your computer, note that you can also use Globus to transfer data to Alpine from your computer.
```

Reply:
Thanks, great points! Committing this in as well.